api/lds.proto

```diff
@@ -66,13 +66,27 @@ message FilterChainMatch {
 
   // If non-empty, an IP address and prefix length to match addresses when the
   // listener is bound to 0.0.0.0/::.
```

Comment:
Your are right, but I would actually argue that in v2 we lose ```findListenerByAddress()``` and do everything via the filter chain match infra. I think this is conceptually much simpler to understand since everything is now in 1 place.